docs/modules/deployment/pages/backup/introduction.adoc

```diff
@@ -0,0 +1,35 @@
+[[backup]]
+= Back up OpenNMS {page-component-title}
+
+This section describes a basic backup process for a default OpenNMS {page-component-title} xref:deployment:core/getting-started.adoc#getting-started-core[installation].
+OpenNMS {page-component-title} binaries, logs, RRD archives, configurations and the database will be backed up.
```

Comment:
Logs are mentioned here, but I don't see a step that includes the logs.  I would argue that log backup is not necessary and should be skipped.

Reply:
> @mmahacek , do you have any additional insights for Marcel? If possible, I'd like to include this in H30, which means we'll need to have it updated an merged by May 6.

If this is going to be included, it should also include RHEL/Cent instructions as well.

Reply:
Correct. The tar command does not backup it. But the symlink to `/var/log/opennms/`. I will remove the mention here.
Thanks!